cmd/frontend/graphqlbackend/search.go

```diff
@@ -234,10 +236,11 @@ func detectSearchType(version string, patternType *string, input string) (query.
 
 // searchResolver is a resolver for the GraphQL type `Search`
 type searchResolver struct {
-	query         query.QueryInfo       // the query, either containing and/or expressions or otherwise ordinary
-	originalQuery string                // the raw string of the original search query
-	pagination    *searchPaginationInfo // pagination information, or nil if the request is not paginated.
-	patternType   query.SearchType
+	query              query.QueryInfo       // the query, either containing and/or expressions or otherwise ordinary
+	originalQuery      string                // the raw string of the original search query
+	pagination         *searchPaginationInfo // pagination information, or nil if the request is not paginated.
+	patternType        query.SearchType
+	versionContextName *string
```

Comment:
Any particular reason to have the `Name` suffix?

```suggestion
	versionContext *string
```